the output of setlocale is quite opaque and segfaults in portable libstdc++ locale() implementation later, so just use current
--
diff --git a/src/musikcube/app/model/DirectoryAdapter.cpp b/src/musikcube/app/model/DirectoryAdapter.cpp
index 6108cde..2f4d1bc 100755
--- a/src/musikcube/app/model/DirectoryAdapter.cpp
+++ b/src/musikcube/app/model/DirectoryAdapter.cpp
@@ -133,7 +133,7 @@ static void buildDirectoryList(
         std::sort(
             target.begin(),
             target.end(),
-            std::locale(setlocale(LC_ALL, nullptr)));
+            std::locale(""));
     }
     catch (...) {
         std::sort(target.begin(), target.end());
